src/unistd.rs

```diff
@@ -1644,8 +1644,18 @@ mod linux {
         Errno::result(res).map(drop)
     }
 
+    /// Replaces the current process image with a new one.
+    /// ([see exec(3)](http://man7.org/linux/man-pages/man3/exec.3.html))
+    ///
+    /// * `filename`: filename of executable to run
+    /// * `args`: array of arguments as strings; first must be the filename
+    /// * `env`: array of environment variables for the new execution environment
+    /// * returns: Ok or libc error code.
+    ///
+    /// An error is returned if the executable file does not exist, cannot be accessed, or 
```

Comment:
There is a special section called `Errors` that you should use for these kinds of things. See [the book](https://doc.rust-lang.org/book/first-edition/documentation.html#special-sections).